### PR TITLE
create todo for front

### PR DESCRIPTION
**Pull request by ViacheslavKechkin**
Hello. Please let me do the merge)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  <script src="https://cdn.jsdelivr.net/lodash/4.10.0/lodash.js"></script>
+  <script src="script.js"></script>
+</head>
+
+<body>
+  <h1>My ToDo</h1>
+  <input type="text" id="add-task">
+  <button onclick="onClickButton()">Add Task</button>
+  <div id="content-page"></div>
+
```
Comment (NataliaShchelkunova) [inline index.html]: remove

File: script.js [hunk 2]
```diff
@@ -0,0 +1,234 @@
+
+//получаю данный из localStorage если их там нет, получаю пустой массив
+let allTasks = JSON.parse(localStorage.getItem('tasks')) || [];
```
Comment (NataliaShchelkunova) [inline script.js]: remove. why u use localStorage?

File: script.js [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+let valueInput = '';
+let input = null;
+let activeEditTask = null;
+
+//с помощью async сделал асинхронную функцию 
+//она будет ждать пока придет ответ с сервера http://
+window.onload = async function init() {
```
Comment (NataliaShchelkunova) [inline script.js]: es6

File: script.js [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+  input = document.getElementById('add-task');
+  //инпуту добавили слушатель изменений
+  input.addEventListener('change', updateValue);
+  //функция ждет ответ с сервера (fetch - метод для сетевого запроса) запрос на получения данных
+  const response = await fetch('http://localhost:8000/allTasks', {
+    method: 'GET'
+  });
+  //как только получу ответ от response, беру данные и приобразовываю данные в json формат 
+  let result = await response.json();
+  console.log('result', result);
```
Comment (NataliaShchelkunova) [inline script.js]: remove

File: script.js [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+  //перезаписываю полученые данные в allTasks (data потому что данные приходят в свойство data у result)
+  allTasks = result.data;
+  console.log('GET allTasks', allTasks);
```
Comment (NataliaShchelkunova) [inline script.js]: remove

File: script.js [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+  //render делаю потому что получил данные из locakStorage и нужно их отобразить
+  render();
+}
+
+
+onClickButton = async () => {
+
```
Comment (NataliaShchelkunova) [inline script.js]: remove empty line

File: script.js [hunk 7]
```diff
@@ -0,0 +1,234 @@
...
+  if (input.value != '') {
+    allTasks.push({
+      text: valueInput,
+      isCheck: false
+    });
```
Comment (NataliaShchelkunova) [inline script.js]: why u push ???????

File: script.js [hunk 8]
```diff
@@ -0,0 +1,234 @@
...
+      body: JSON.stringify({
+        text: valueInput,
+        isCheck: false
+      })
+    });
+    //как только получу ответ от response, беру данные и приобразовываю данные в json формат 
+    let result = await response.json();
+    //перезаписываю полученые данные в allTasks (data потому что данные приходят в свойство data у result)
+    allTasks = result.data;
+    console.log('result', allTasks);
```
Comment (NataliaShchelkunova) [inline script.js]: remove all console.logs

File: script.js [hunk 9]
```diff
@@ -0,0 +1,234 @@
...
+    //сохраняю данные в locakStorage когда произошли какие либо изменения
+    //в массиве allTasks
+    localStorage.setItem('tasks', JSON.stringify(allTasks));
+    //чистим что бы данные не дублировались
```
Comment (NataliaShchelkunova) [inline script.js]: why???????

File: script.js [hunk 10]
```diff
@@ -0,0 +1,234 @@
...
+    render();
+  }
+}
+//получаем значение изменения в инпуте
+updateValue = (event) => {
+  valueInput = event.target.value;
+}
+
+// функция рендора задач
+render = () => {
```
Comment (NataliaShchelkunova) [inline script.js]: const

File: script.js [hunk 11]
```diff
@@ -0,0 +1,234 @@
...
+    const container = document.createElement('div');
+    container.id = `task-${index}`;
+    container.className = 'task-container';
+
+    //Добавляю чекбокс в контейнер
+    const checkbox = document.createElement('input');
+    checkbox.type = 'checkbox';
+    checkbox.checked = item.isCheck;
+    checkbox.className = 'task-checkbox';
+    checkbox.onchange = function () {
```
Comment (NataliaShchelkunova) [inline script.js]: es6

File: script.js [hunk 12]
```diff
@@ -0,0 +1,234 @@
...
+      onChangeCheckbox(index);
+    };
+    container.appendChild(checkbox);
```
Comment (NataliaShchelkunova) [inline script.js]: do u always render checkbox ?

File: script.js [hunk 13]
```diff
@@ -0,0 +1,234 @@
...
+      text.className = item.isCheck ? 'text-task done-text' : 'text-task';
+      container.appendChild(text);
+    }
+
+    //проверка если задача выполнена то сюда не заходить
+    if (!item.isCheck) {
+      if (index === activeEditTask) {
+        const imageDone = document.createElement('img');
+        imageDone.src = 'img/done.png';
+        imageDone.onclick = function () {
```
Comment (NataliaShchelkunova) [inline script.js]: es6

File: script.js [hunk 14]
```diff
@@ -0,0 +1,234 @@
...
+          doneEditTask();
+        };
+        container.appendChild(imageDone);
+      } else {
+        //Добавляю картинку для изменения задачи
+        const imageEdit = document.createElement('img');
+        imageEdit.src = 'img/edit.png';
+        imageEdit.onclick = function () {
```
Comment (NataliaShchelkunova) [inline script.js]: es6

File: script.js [hunk 15]
```diff
@@ -0,0 +1,234 @@
...
+          render();
+        }
+        container.appendChild(imageEdit);
+      }
+
+    }
+    //Добавляю картинку для удаления задачи
+    const imageDelete = document.createElement('img');
+    imageDelete.src = 'img/delete.png';
+    imageDelete.onclick = function () {
```
Comment (NataliaShchelkunova) [inline script.js]: es6

File: script.js [hunk 16]
```diff
@@ -0,0 +1,234 @@
...
+    }
+    container.appendChild(imageDelete);
+
+    //Добавляю сформированный контейре в контент на странице
+    content.appendChild(container);
+  });
+}
+
+//функция изменения состаяния чекбокса
+onChangeCheckbox = async (index) => {
```
Comment (NataliaShchelkunova) [inline script.js]: const

File: script.js [hunk 17]
```diff
@@ -0,0 +1,234 @@
...
+  allTasks[index].isCheck = !allTasks[index].isCheck;
```
Comment (NataliaShchelkunova) [inline script.js]: destructurisation

File: script.js [hunk 18]
```diff
@@ -0,0 +1,234 @@
...
+      //описываем в каком формате отдаем данные и в каком должен принимать их сервер
+      'Content-Type': 'application/json;charset=utf-8',
+      //указываем разрешение на локальное приложение
+      'Access-Control-Allow-Origin': '*'
+    },
+    // в body информация о новом таске
+    body: JSON.stringify({
+      id: allTasks[index].id,
+      text: allTasks[index].text,
+      isCheck: allTasks[index].isCheck
```
Comment (NataliaShchelkunova) [inline script.js]: destructurisation

File: script.js [hunk 19]
```diff
@@ -0,0 +1,234 @@
...
+    })
+  });
+  //как только получу ответ от response, беру данные и приобразовываю данные в json формат 
+  let result = await response.json();
+  //перезаписываю полученые данные в allTasks (data потому что данные приходят в свойство data у result)
+  allTasks = result.data;
+  //сохраняю данные в locakStorage когда произошли какие либо изменения
+  //в массиве allTasks
+  localStorage.setItem('tasks', JSON.stringify(allTasks));
```
Comment (NataliaShchelkunova) [inline script.js]: why???

File: script.js [hunk 20]
```diff
@@ -0,0 +1,234 @@
...
+  render();
+}
+//функция удаления задачи
+deleteTask = async (index, itemIdDel) => {
```
Comment (NataliaShchelkunova) [inline script.js]: const

File: script.js [hunk 21]
```diff
@@ -0,0 +1,234 @@
...
+  allTasks.splice(index, 1);
```
Comment (NataliaShchelkunova) [inline script.js]: why?

File: script.js [hunk 22]
```diff
@@ -0,0 +1,234 @@
...
+    },
+  });
+  //как только получу ответ от response, беру данные и приобразовываю данные в json формат 
+  let result = await response.json();
+  //перезаписываю полученые данные в allTasks (data потому что данные приходят в свойство data у result)
+  allTasks = result.data;
+
+  //сохраняю данные в locakStorage когда произошли какие либо изменения
+  //в массиве allTasks
+  localStorage.setItem('tasks', JSON.stringify(allTasks));
```
Comment (NataliaShchelkunova) [inline script.js]: why?

File: script.js [hunk 23]
```diff
@@ -0,0 +1,234 @@
...
+  render();
+}
+
+updateTaskText = async (event) => {
+  allTasks[activeEditTask].text = event.target.value;
+  //функция ждет ответ с сервера (fetch - метод для сетевого запроса) запрос на изменение данных
+  const response = await fetch('http://localhost:8000/updateTask', {
+    method: 'PATCH',
+    headers: {
```
Comment (NataliaShchelkunova) [inline script.js:151]: const

File: script.js [hunk 24]
```diff
@@ -0,0 +1,234 @@
...
+      //описываем в каком формате отдаем данные и в каком должен принимать их сервер
+      'Content-Type': 'application/json;charset=utf-8',
+      //указываем разрешение на локальное приложение
+      'Access-Control-Allow-Origin': '*'
+    },
+    // в body информация о новом таске
+    body: JSON.stringify({
+      id: allTasks[activeEditTask].id,
+      text: allTasks[activeEditTask].text,
+      isCheck: false
```
Comment (NataliaShchelkunova) [inline script.js]: destructurisation, isCheck: !allTasks[activeEditTask].isCheck And destructurisation

File: script.js [hunk 25]
```diff
@@ -0,0 +1,234 @@
...
+    })
+  });
+  //как только получу ответ от response, беру данные и приобразовываю данные в json формат 
+  let result = await response.json();
+  //перезаписываю полученые данные в allTasks (data потому что данные приходят в свойство data у result)
+  allTasks = result.data;
+  console.log('PATCH allTasks', allTasks);
```
Comment (NataliaShchelkunova) [inline script.js]: remove all console log

File: script.js [hunk 26]
```diff
@@ -0,0 +1,234 @@
...
+
+  //сохраняю данные в locakStorage когда произошли какие либо изменения
+  //в массиве allTasks
+  localStorage.setItem('tasks', JSON.stringify(allTasks));
```
Comment (NataliaShchelkunova) [inline script.js]: remove

File: script.js [hunk 27]
```diff
@@ -0,0 +1,234 @@
...
+  //перезаписываю полученые данные в allTasks (data потому что данные приходят в свойство data у result)
+  allTasks = result.data;
+
+  //сохраняю данные в locakStorage когда произошли какие либо изменения
+  //в массиве allTasks
+  localStorage.setItem('tasks', JSON.stringify(allTasks));
+  render();
+}
+
+updateTaskText = async (event) => {
```
Comment (NataliaShchelkunova) [inline script.js]: const

File: script.js [hunk 28]
```diff
@@ -0,0 +1,234 @@
...
+  allTasks = result.data;
+  console.log('PATCH allTasks', allTasks);
+
+  //сохраняю данные в locakStorage когда произошли какие либо изменения
+  //в массиве allTasks
+  localStorage.setItem('tasks', JSON.stringify(allTasks));
+  render();
+}
+
+doneEditTask = () => {
```
Comment (NataliaShchelkunova) [inline script.js]: const

File: style.css [hunk 29]
```diff
@@ -0,0 +1,54 @@
+
```
Comment (NataliaShchelkunova) [inline style.css]: remove empty line

File: style.css [hunk 30]
```diff
@@ -0,0 +1,54 @@
...
+}
+button {
+  margin: 10px 0 10px 0;
+  width: 185px;
+  height: 30px;
+  background-color: rgb(255, 153, 0);
+  border: none;
+  cursor: pointer;
+  box-shadow: 0 0 5px gray;
+}
```
Comment (NataliaShchelkunova) [inline style.css:16]: add empty line

File: style.css [hunk 31]
```diff
@@ -0,0 +1,54 @@
...
+h1 {
+  font-size: 45px;
+  text-align: center;
+  color: rgb(231, 99, 99);
+}
```
Comment (NataliaShchelkunova) [inline style.css:22]: add empty line

File: style.css [hunk 32]
```diff
@@ -0,0 +1,54 @@
...
+img {
+  height: 40px;
+  width: 40px;
+  margin-left: 10px;
+  cursor: pointer;
+}
```
Comment (NataliaShchelkunova) [inline style.css:29]: add empty line

File: style.css [hunk 33]
```diff
@@ -0,0 +1,54 @@
...
+#add-task {
+  height: 30px;
+  color: blue;
+}
```
Comment (NataliaShchelkunova) [inline style.css:34]: add empty line

File: style.css [hunk 34]
```diff
@@ -0,0 +1,54 @@
...
+.task-checkbox {
+  width: 25px;
+  height: 25px;
+}
```
Comment (NataliaShchelkunova) [inline style.css:39]: add empty line

File: style.css [hunk 35]
```diff
@@ -0,0 +1,54 @@
...
+#content-page {
+  display: flex;
+  flex-direction: column;
+  align-items: left;
+}
+
+.task-container {
+  display: flex;
+  align-items: center;
+}
```
Comment (NataliaShchelkunova) [inline style.css:55]: add empty line

File: style.css [hunk 36]
```diff
@@ -0,0 +1,54 @@
+
+body {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
```
Comment (NataliaShchelkunova) [inline style.css:6]: add empty line

File: script.js [hunk 37]
```diff
@@ -0,0 +1,170 @@
...
+  const response = await fetch('http://localhost:8000/updateTask', {
+    method: 'PATCH',
+    headers: {
+      'Content-Type': 'application/json;charset=utf-8',
+      'Access-Control-Allow-Origin': '*'
+    },
+    body: JSON.stringify({
+      id: allTasks[activeEditTask].id,
+      text: allTasks[activeEditTask].text,
+      isCheck: false
```
Comment (NataliaShchelkunova) [inline script.js]: destructurisation

File: script.js [hunk 38]
```diff
@@ -0,0 +1,166 @@
...
+  let { id, text } = newElArr;
+  text = event.target.value;
+  const response = await fetch('http://localhost:8000/updateTask', {
+    method: 'PATCH',
+    headers: {
+      'Content-Type': 'application/json;charset=utf-8',
+      'Access-Control-Allow-Origin': '*'
+    },
+    body: JSON.stringify({
+      id: id,
```
Comment (NataliaShchelkunova) [inline script.js]: only id, text and is check. why u send is check false if it false now?

File: script.js (hunk 6 above)
Comment (NataliaShchelkunova) [reply script.js]: !!!!!!!

File: script.js [hunk 39]
```diff
@@ -0,0 +1,166 @@
...
+      text.innerText = item.text;
+      text.className = item.isCheck ? 'text-task done-text' : 'text-task';
+      container.appendChild(text);
+    }
+
+    if (!item.isCheck) {
+      if (index === activeEditTask) {
+        const imageDone = document.createElement('img');
+        imageDone.src = 'img/done.png';
+        imageDone.onclick = () => {
```
Comment (NataliaShchelkunova) [inline script.js]: one line

File: script.js [hunk 40]
```diff
@@ -0,0 +1,166 @@
...
+          activeEditTask = index;
+          render();
+        }
+        container.appendChild(imageEdit);
+      }
+    }
+    const imageDelete = document.createElement('img');
+    imageDelete.src = 'img/delete.png';
+    imageDelete.onclick = () => {
+      let itemIdDel = item.id;
```
Comment (NataliaShchelkunova) [inline script.js]: why let?

File: script.js [hunk 41]
```diff
@@ -0,0 +1,166 @@
...
+      deleteTask(index, itemIdDel);
+    }
+    container.appendChild(imageDelete);
+    content.appendChild(container);
+  });
+}
+
+const onChangeCheckbox = async (index) => {
+  let elArr = allTasks[index];
```
Comment (NataliaShchelkunova) [inline script.js]: what it is elArr?

File: script.js [hunk 42]
```diff
@@ -0,0 +1,166 @@
...
+  isCheck = !isCheck;
+
+  const response = await fetch('http://localhost:8000/updateTask', {
+    method: 'PATCH',
+    headers: {
+      'Content-Type': 'application/json;charset=utf-8',
+      'Access-Control-Allow-Origin': '*'
+    },
+    body: JSON.stringify({
+      id: id,
```
Comment (NataliaShchelkunova) [inline script.js]: only id text and isCheck

File: script.js [hunk 43]
```diff
@@ -0,0 +1,166 @@
...
+    },
+  });
+  let result = await response.json();
+  allTasks = result.data;
+  render();
+}
+
+const updateTaskText = async (event) => {
+  let newElArr = allTasks[activeEditTask];
+  let { id, text } = newElArr;
```
Comment (NataliaShchelkunova) [inline script.js]: why let?

File: script.js [hunk 44]
```diff
@@ -0,0 +1,166 @@
...
+      'Access-Control-Allow-Origin': '*'
+    },
+  });
+  let result = await response.json();
+  allTasks = result.data;
+  render();
+}
+
+const updateTaskText = async (event) => {
+  let newElArr = allTasks[activeEditTask];
```
Comment (NataliaShchelkunova) [inline script.js]: why let? What it is  newElArr?

File: script.js [hunk 45]
```diff
@@ -0,0 +1,166 @@
...
+      deleteTask(index, itemIdDel);
+    }
+    container.appendChild(imageDelete);
+    content.appendChild(container);
+  });
+}
+
+const onChangeCheckbox = async (index) => {
+  let elArr = allTasks[index];
+  let { isCheck, text, id } = elArr;
```
Comment (NataliaShchelkunova) [inline script.js]: why let?

File: script.js (hunk 41 above)
Comment (NataliaShchelkunova) [reply script.js]: why let?

File: script.js (hunk 43 above)
Comment (ViacheslavKechkin) [reply script.js]: if I use const - TypeError: Assignment to constant variable.

File: script.js [hunk 46]
```diff
@@ -0,0 +1,155 @@
...
+        isCheck: false
+      })
+    });
+    let result = await response.json();
+    allTasks = result.data;
+    valueInput = '';
+    input.value = '';
+    render();
+  }
+}
```
Comment (NataliaShchelkunova) [inline script.js:36]: add empty line

File: script.js [hunk 47]
```diff
@@ -0,0 +1,155 @@
...
+  allTasks.map((item, index) => {
+    const container = document.createElement('div');
+    container.id = `task-${index}`;
+    container.className = 'task-container';
+
+    const checkbox = document.createElement('input');
+    checkbox.type = 'checkbox';
+    checkbox.checked = item.isCheck;
+    checkbox.className = 'task-checkbox';
+    checkbox.onchange = () => { onChangeCheckbox(index); render(); };
```
Comment (NataliaShchelkunova) [inline script.js]: ? what is it

File: script.js (hunk 12 above)
Comment (NataliaShchelkunova) [reply script.js]: ???????

File: script.js [hunk 48]
```diff
@@ -0,0 +1,155 @@
...
+      text.innerText = item.text;
+      text.className = item.isCheck ? 'text-task done-text' : 'text-task';
+      container.appendChild(text);
+    }
+
+    if (!item.isCheck) {
+      if (index === activeEditTask) {
+        const imageDone = document.createElement('img');
+        imageDone.src = 'img/done.png';
+        imageDone.onclick = () => { doneEditTask(); };
```
Comment (NataliaShchelkunova) [inline script.js]: read lern js

File: script.js [hunk 49]
```diff
@@ -0,0 +1,155 @@
...
+        container.appendChild(imageDone);
+      } else {
+        const imageEdit = document.createElement('img');
+        imageEdit.src = 'img/edit.png';
+        imageEdit.onclick = () => { activeEditTask = index; render(); }
```
Comment (NataliaShchelkunova) [inline script.js]: what is it

File: script.js [hunk 50]
```diff
@@ -0,0 +1,155 @@
...
+        container.appendChild(imageEdit);
+      }
+    }
```
Comment (NataliaShchelkunova) [inline script.js:88]: add empty line

File: script.js (hunk 42 above)
Comment (NataliaShchelkunova) [reply script.js]: !!!!!!!!!!!

File: script.js (hunk 38 above)
Comment (NataliaShchelkunova) [reply script.js]: !!!!!

File: script.js (hunk 12 above)
Comment (vykechkin) [reply script.js:58]: I add render

File: script.js (hunk 47 above)
Comment (NataliaShchelkunova) [reply script.js]: ?????

File: script.js (hunk 12 above)
Comment (NataliaShchelkunova) [reply script.js]: ??????

File: script.js [hunk 51]
```diff
@@ -0,0 +1,146 @@
...
+    const container = document.createElement('div');
+    container.id = `task-${index}`;
+    container.className = 'task-container';
+
+    const checkbox = document.createElement('input');
+    checkbox.type = 'checkbox';
+    checkbox.checked = item.isCheck;
+    checkbox.className = 'task-checkbox';
+    checkbox.onchange = () => onChangeCheckbox(index);
+    container.appendChild(checkbox);
```
Comment (NataliaShchelkunova) [inline script.js]: do you always need to render the checkbox?

File: script.js [hunk 52]
```diff
@@ -0,0 +1,146 @@
...
+        const imageEdit = document.createElement('img');
+        imageEdit.src = 'img/edit.png';
+        imageEdit.onclick = () => { activeEditTask = index; render() }
+        container.appendChild(imageEdit);
+      }
+    }
+
+    const imageDelete = document.createElement('img');
+    imageDelete.src = 'img/delete.png';
+    imageDelete.onclick = () => { let itemIdDel = item.id; deleteTask(index, itemIdDel) };
```
Comment (NataliaShchelkunova) [inline script.js]: what it is? pls use different lines for function body

File: script.js [hunk 53]
```diff
@@ -0,0 +1,146 @@
...
+    container.appendChild(imageDelete);
+    content.appendChild(container);
+  });
+}
+
+const onChangeCheckbox = async (index) => {
+  const elementArr = allTasks[index];
```
Comment (NataliaShchelkunova) [inline script.js]: do you really need this variable? pls using direct assignment

File: script.js [hunk 54]
```diff
@@ -0,0 +1,146 @@
...
+  let { isCheck, text, id } = elementArr;
```
Comment (NataliaShchelkunova) [inline script.js]: = allTasks[index]; without elementArr

File: script.js [hunk 55]
```diff
@@ -0,0 +1,146 @@
...
+  isCheck = !isCheck;
+
+  const response = await fetch('http://localhost:8000/updateTask', {
+    method: 'PATCH',
+    headers: {
+      'Content-Type': 'application/json;charset=utf-8',
+    },
+    body: JSON.stringify({
+      id: id,
```
Comment (NataliaShchelkunova) [inline script.js]: why u don't fix my comments????

File: script.js [hunk 56]
```diff
@@ -0,0 +1,146 @@
...
+  let { id, text } = elementArr;
+  text = event.target.value;
+  const response = await fetch('http://localhost:8000/updateTask', {
+    method: 'PATCH',
+    headers: {
+      'Content-Type': 'application/json;charset=utf-8',
+    },
+    body: JSON.stringify({
+      id: id,
+      text: text,
```
Comment (NataliaShchelkunova) [inline script.js]: why u don't fix my comments?

File: script.js [hunk 57]
```diff
@@ -0,0 +1,148 @@
...
+    if (!item.isCheck) {
+      if (index === activeEditTask) {
+        const imageDone = document.createElement('img');
+        imageDone.src = 'img/done.png';
+        imageDone.onclick = () => doneEditTask();
+        container.appendChild(imageDone);
+      } else {
+        const imageEdit = document.createElement('img');
+        imageEdit.src = 'img/edit.png';
+        imageEdit.onclick = () => { activeEditTask = index; render() }
```
Comment (NataliaShchelkunova) [inline script.js]: code stylr

File: script.js [hunk 58]
```diff
@@ -0,0 +1,148 @@
...
+    },
+  });
+  let result = await response.json();
+  allTasks = result.data;
+  render();
+}
+
+const updateTaskText = async (event) => {
+  const elementArr = allTasks[activeEditTask];
+  let { id, text } = elementArr;
```
Comment (NataliaShchelkunova) [inline script.js]: pls fix

File: style.css (hunk 35 above)
Comment (NataliaShchelkunova) [reply style.css:55]: !!!!!

File: script.js (hunk 40 above)
Comment (NataliaShchelkunova) [reply script.js]: !!!!!!

**Pull request merged**